plugins/services/src/js/containers/framework-configuration/FrameworkConfigurationContainer.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+
+import UniversePackage from "#SRC/js/structs/UniversePackage";
+
+import Framework from "#PLUGINS/services/src/js/structs/Framework";
+import { routerShape } from "react-router";
+import FrameworkConfigurationReviewScreen
+  from "#SRC/js/components/FrameworkConfigurationReviewScreen";
+import Loader from "src/js/components/Loader";
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
+import { COSMOS_SERVICE_DESCRIBE_CHANGE } from "#SRC/js/constants/EventTypes";
+
+const METHODS_TO_BIND = [
+  "reorderResolvedOptions",
+  "handleEditClick",
+  "onCosmosPackagesStoreServiceDescriptionSuccess"
+];
+
+class FrameworkConfigurationContainer extends React.Component {
```

Comment:
Ins't this going to be confused by https://github.com/dcos/dcos-ui/blob/ahoskins/feat/DCOS_OSS-1789-framework-configuration-tab/src/js/components/FrameworkConfigurationReviewScreen.js ?

Reply:
Ooops, yes good catch. Will fix that naming.